spec/views/job_runs/show.html.erb_spec.rb

```diff
@@ -4,13 +4,14 @@
   before do
     assign(:job_run, job_run)
     allow(job_run).to receive(:progress_log_file).and_return(actual_file)
-    allow(job_run).to receive(:output_location).and_return(actual_file)
   end
 
```

Comment:
same as above, pre-assembly jobs don't have this field set anymore, let's make this test better reflect reality to prevent future problems if we add more tests